### ISSUE TITLE
picard.jar file corrupted for v2.8.1

### ISSUE THREAD

**Issue by pwanj** [broadinstitute/picard#1413]
Hello,

The picard.jar for v2.8.1 throws the following error:

Error: Invalid or corrupt jarfile picard.jar

I downloaded it from https://github.com/broadinstitute/picard/releases

Please can you direct me to the correct picard.jar file for this version. Thanks

**Comment by lbergelson** [comment]
I just downloaded it and tried it myself and it seems to run fine.  Maybe your download was corrupted?

Maybe check the md5 of your download and make sure it's correct?
```
md5 picard.jar
MD5 (picard.jar) = 5d0bc87e26797be66bd7d9dd8660bec3
```

Also, picard 2.8.1 requires java 8.  Is that what you are running?

**Comment by pwanj** [comment]
Yes, you are correct. The file was corrupted. It works now. Sorry for the inconvenience. Thank you for the help.

**Comment by elbakrifz** [comment]
when I used this command :  java -jar picard.jar CreateSequenceDictionary/
I got : Error: Invalid or corrupt jarfile picard.jar
 and when I check the md5 of download I got : MD5 (/Users/mac/java/picard.jar) = d41d8cd98f00b204e9800998ecf8427e
is correct ??

**Comment by elbakrifz** [comment]
when I run : 
ava -jar /Users/mac/Downloads/picard-2.jar CreateSequenceDictionary REFERENCE=/Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.fasta OUTPUT=/Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.dict
I got : 
INFO	2020-06-17 19:26:30	CreateSequenceDictionary	

********** NOTE: Picard's command line syntax is changing.
**********
********** For more information, please see:
********** https://github.com/broadinstitute/picard/wiki/Command-Line-Syntax-Transition-For-Users-(Pre-Transition)
**********
********** The command line looks like this in the new syntax:
**********
**********    CreateSequenceDictionary -REFERENCE /Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.fasta -OUTPUT /Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.dict
**********


19:26:30.984 INFO  NativeLibraryLoader - Loading libgkl_compression.dylib from jar:file:/Users/mac/Downloads/picard-2.jar!/com/intel/gkl/native/libgkl_compression.dylib
[Wed Jun 17 19:26:31 WEST 2020] CreateSequenceDictionary OUTPUT=/Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.dict REFERENCE=/Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.fasta    TRUNCATE_NAMES_AT_WHITESPACE=true NUM_SEQUENCES=2147483647 VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
[Wed Jun 17 19:26:31 WEST 2020] Executing as mac@Air-de-mac on Mac OS X 10.15.3 x86_64; OpenJDK 64-Bit Server VM 11.0.1+13-LTS; Deflater: Intel; Inflater: Intel; Provider GCS is not available; Picard version: 2.23.0
[Wed Jun 17 19:26:31 WEST 2020] picard.sam.CreateSequenceDictionary done. Elapsed time: 0,01 minutes.
Runtime.totalMemory()=134217728
To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp
Exception in thread "main" picard.PicardException: File /Volumes/major/DATASETS/LmjFwholegenome_20070731_V5.2.dict not found
	at picard.sam.CreateSequenceDictionary.doWork(CreateSequenceDictionary.java:248)
	at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:305)
	at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:103)
	at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:113)

I didn't find the file.dict ?
